3.0/vector_tile.proto

```diff
@@ -27,6 +27,31 @@ message Tile {
                 extensions 8 to max;
         }
 
+        message Dimension {
+            // is_elevation can only be used once for all dimensions in the layer
+            optional bool is_elevation = 2;
+
+            // Formula for values in dimension:
+            // value = global_offset + scale * (delta_encoded_value + offset)
+
+            // Individual offset for this specific tile
+            optional sint64 offset = 3;
+
+            // Only one of scale_int or scale_double may appear.
+            // If neither scale_int nor scale_double appears, scale = 1.0.
+            optional sint64 scale_int = 4;
+            optional double scale_double = 8; 
+
+            // Only one of global_offset_int or global_offset_double may appear.
+            // If neither global_offset_int nor global_offset_double appears,
+            // global_offset = 0.0.
+            optional sint64 global_offset_int = 5;
+            optional double global_offset_double = 9;
```

Comment:
Is the int/double thing independent for scale and global_offset, or can you only choose for both of them to be int or both of them to be double?